### DISCUSSION TITLE
macOS 云盘上传歌曲总是失败报错

### DISCUSSION THREAD

**Discussion by Mx-Iris** [qier222/YesPlayMusic#1185] [Q&A]
因为Mac官方版网易云不提供上传云盘的功能，看到这个软件有上传按钮，但是我从来没上传成功过
macOS版本：12.1
软件版本：0.4.3
<img width="260" alt="Xnip 2022-01-05_16-09-44@2x" src="https://user-images.githubusercontent.com/61279231/148182742-fe6b5a0f-9e81-49f4-b1b4-ba7c5721a160.png">

**Comment by pan93412** [comment]
413 是檔案太大的意思。
